### PR TITLE
Cleanup axf shader

### PR DESCRIPTION
**Pull request by sebastienlagarde**
### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?

---
### Release Notes
Cleanup AxF code, remove debug code and convert to our coding convention.
Remove BaseColor that isn't used.

---
### Testing status
**Katana Tests**: https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=cleanup-axf-shader&automation-tools_branch=add-platform-filter&unity_branch=trunk#

**Manual Tests**:  Check on local AxF test scene

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Editor/Material/AxF/AxFUI.cs [hunk 1]
```diff
@@ -405,16 +382,16 @@ protected unsafe override void MaterialPropertiesGUI(Material _material)
                     m_CarPaint2_BRDFColorMapScale.floatValue = EditorGUILayout.FloatField(Styles.BRDFColorMapScaleText, m_CarPaint2_BRDFColorMapScale.floatValue);
 
                     m_MaterialEditor.TexturePropertySingleLine(Styles.BTFFlakesMapText, m_CarPaint2_BTFFlakeMap);
-//EditorGUILayout.LabelField( "Texture Dimension = " + m_CarPaint_BTFFlakesMap_sRGB.textureDimension );
-//EditorGUILayout.LabelField( "Texture Format = " + m_CarPaint_BTFFlakesMap_sRGB.textureValue. );
+                    //EditorGUILayout.LabelField( "Texture Dimension = " + m_CarPaint_BTFFlakesMap_sRGB.textureDimension );
```
Comment (JulienIgnace-Unity) [inline com.unity.render-pipelines.high-definition/Editor/Material/AxF/AxFUI.cs]: Is this commented out code going to be used at some point?

File: com.unity.render-pipelines.high-definition/Runtime/Material/AxF/AxF.hlsl [hunk 2]
```diff
@@ -377,7 +377,7 @@ PreLightData    GetPreLightData(float3 viewWS_Clearcoat, PositionInputs posInput
     preLightData.viewWS_UnderCoat = viewWS_Clearcoat;   // Save original view before optional refraction by clearcoat
 
 
-    // ==============================================================================
+                                                        //-----------------------------------------------------------------------------
```
Comment (JulienIgnace-Unity) [inline com.unity.render-pipelines.high-definition/Runtime/Material/AxF/AxF.hlsl]: Indentation

File: com.unity.render-pipelines.high-definition/Runtime/Material/AxF/AxFData.hlsl [hunk 3]
```diff
@@ -45,29 +45,18 @@ void GetSurfaceAndBuiltinData(FragInputs input, float3 V, inout PositionInputs p
...
+    float2 UV0 = input.texCoord0.xy * float2(_MaterialTilingU, _MaterialTilingV);
 
     //-----------------------------------------------------------------------------
     // _AXF_BRDF_TYPE_SVBRDF
     //-----------------------------------------------------------------------------
 
 #ifdef _AXF_BRDF_TYPE_SVBRDF
 
-    surfaceData.diffuseColor = SAMPLE_TEXTURE2D(_SVBRDF_DiffuseColorMap, sampler_SVBRDF_DiffuseColorMap, UV0).xyz * _BaseColor.xyz;
+    surfaceData.diffuseColor = SAMPLE_TEXTURE2D(_SVBRDF_DiffuseColorMap, sampler_SVBRDF_DiffuseColorMap, UV0).xyz;
```
Comment (JulienIgnace-Unity) [inline com.unity.render-pipelines.high-definition/Runtime/Material/AxF/AxFData.hlsl]: We don't ever multiply by base color anymore?

Comment (sebastienlagarde) [reply com.unity.render-pipelines.high-definition/Runtime/Material/AxF/AxFData.hlsl]: This is the purpose of the PR :)

File: com.unity.render-pipelines.high-definition/Editor/Material/AxF/AxFUI.cs (hunk 1 above)
Comment (sebastienlagarde) [reply com.unity.render-pipelines.high-definition/Editor/Material/AxF/AxFUI.cs]: No idea, I let it here for now. I don't know anything about AxF spec

**Pull request merged**